server/src/database/db_connection.js

```diff
@@ -1,24 +1,25 @@
-// Add code below to connect to your database
-const pg = require('pg');
-const Pool = pg.Pool;
-// const {POOL} = require("pg");
-const url = require('url');
-require('dotenv').config();
+const { Pool } = require("pg");
+const url = require("url");
+require("dotenv").config();
 
-if (!process.env.DB_URL) throw new Error('Envinroment: variable DB_URL must be set');
+let DB_URL = process.env.DB_URL;
```

Comment:
why did you remove the if?

Reply:
Hi, I removed it because it failed for undefined  process.env.DB_URL
and I replaced it with: 
DB_URL = process.env.TEST_DB_URL;
if (!DB_URL) throw new Error("Enviroment variable DB_URL must be set");